daliuge-translator/dlg/dropmake/pg_generator.py

```diff
@@ -816,7 +816,7 @@ def make_single_drop(self, iid="0", **kwargs):
 
     @staticmethod
     def str_to_bool(value, default_value=False):
-        res = True if value in ["1", "true", "yes"] else default_value
+        res = True if value in ["1", "true", "True", "yes"] else default_value
```

Comment:
I'm not sure why this the translator receives a capital boolean when it is lowecase in eagle, my best guess is a cast to a boolean and python serializes to python literal instead of json literal.

Reply:
Yes, I think you're right, that literal in python is "True" and in javascript is "true". I'm not sure where it gets serialised, but that seems likely to be the issue.